test/controllers/ObligedEntityPdfControllerSpec.scala

```diff
@@ -105,6 +105,10 @@ class ObligedEntityPdfControllerSpec extends SpecBase {
         val result = route(application, request).value
 
         status(result) mustEqual INTERNAL_SERVER_ERROR
+        // added for testing the DDCE-5603 for custom error template
```

Comment:
I can see most views are tested using this sort of format e.g. UnauthorisedControllerSpec.scala: 
      contentAsString(result) mustEqual
        view()(request, messages).toString

Reply:
yeah i check for the contains on the page, so I've used this lines 

contentAsString(result) contains